etcdserver/v3_server.go

```diff
@@ -222,6 +222,9 @@ func (s *EtcdServer) LeaseGrant(ctx context.Context, r *pb.LeaseGrantRequest) (*
 }
 
 func (s *EtcdServer) LeaseRevoke(ctx context.Context, r *pb.LeaseRevokeRequest) (*pb.LeaseRevokeResponse, error) {
+	// fix: LeaseRevoke may fail to apply when authentication is enabled and upgrading cluster from etcd-3.2 to etcd-3.3
+	// see https://github.com/etcd-io/etcd/issues/11689
+	ctx = s.authStore.WithRoot(ctx)
```

Comment:
I think it's practically works fine for now. But it would be safer to have a mechanism to revert the assigned token after calling `raftRequestOnce()`.

Reply:
I noticed that the simpleToken will expire after 5 minutes. Do you mean that we should delete the token immediately after calling `raftRequestOnce`?

Reply:
Yeah, I thought clearing would be fine without waiting expiring. But it's just for defensive purpose. So it's fine for not to do so.